core/tests/test_floor_sections.model.py

```diff
@@ -0,0 +1,53 @@
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from rest_framework.exceptions import ValidationError
+
+from ..models import OfficeBlock, FloorSection
+
+from core.tests import CoreBaseTestCase
+
+User = get_user_model()
+
+
+class OfficeBlockModelTest(CoreBaseTestCase):
```

Comment:
<kbd>CTRL</kbd>+<kbd>C</kbd>

<kbd>CTRL</kbd>+<kbd>V</kbd>

🤔 `OfficeBlockModelTest` ?

Reply:
Actually, it wasn't I first wrote the test in the under the same class `OfficeBlockModelTest`. And later moved it to another file afterwards.

Reply:
Considering that there's a `OfficeBlockModelTest` testcase class [already](https://github.com/AndelaOSP/art-backend/blob/develop/core/tests/test_building_model.py), I recommend:
- first; rename [this](https://github.com/AndelaOSP/art-backend/blob/develop/core/tests/test_building_model.py) test file to `test_office.py` (this will be the module with all the tests related to the office)
- then, move the test methods for the office floor in this module to `test_office.py`
- refactor

Reply:
Done.